packages/orchestration/src/chain-info.js

```diff
@@ -34,7 +34,6 @@ const knownChains = /** @satisfies {Record<string, ChainInfo>} */ (
             version: 'ics20-1',
           },
           versions: [{ identifier: '', features: ['', ''] }],
-          delay_period: 0n,
```

Comment:
how is delay_period not like the others? I don't see anything that makes it less essential

Reply:
I couldn't find a data source for it and I couldn't find a consumer of it